src/main/scala/no/ndla/articleapi/ArticleApiProperties.scala

```diff
@@ -70,7 +70,18 @@ object ArticleApiProperties extends LazyLogging {
 
   val resourceHtmlEmbedTag = "embed"
 
+  def verify() = {
+    val missingProperties = ArticleApiProps.filter(entry => entry._2.isEmpty).toList
+    if(missingProperties.nonEmpty){
+      missingProperties.foreach(entry => logger.error("Missing required environment variable {}", entry._1))
+
+      logger.error("Shutting down.")
+      System.exit(1)
+    }
+  }
+
   def setProperties(properties: Map[String, Option[String]]) = {
+    verify()
```

Comment:
Dette vil ikke fungere ser det ut som. 
Du kjører `verify` før properties blir satt, så `ArticleApiProps` vil være et tomt map.

I metoden `PropertiesLoader.load()` så har vi funksjonalitet til å logge + exite dersom noe går galt. 
Vil da helst at metoden `setProperties` returnerer en `Success` dersom alle properties er satt korrekt, og en `Failure` dersom noen properties mangler i stedet for at vi har verify som exiter selv.